tests/clip/test_modeling_tf_clip.py

```diff
@@ -367,6 +396,47 @@ def test_model_from_pretrained(self):
             model = TFCLIPTextModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
 
+    @slow
+    def test_saved_model_creation_extended(self):
```

Comment:
I think it might be a good idea to put `TFCLIP(Text/Vision)ModelTest` as `TFCoreModelTesterMixin`. And if we do so, probably we don't need to overwrite `test_saved_model_creation_extended` here.

Like
```
class TFBartModelTest(TFModelTesterMixin, TFCoreModelTesterMixin, unittest.TestCase):
```

If we indeed overwrite here, there are some blocks don't really make sense, like `if self.is_encoder_decoder:`.

**This is just a remark, I am happy to leave them as the current version in your PR.**